streams/fr.m3u

```diff
@@ -47,8 +49,16 @@ https://dbmtv.vedge.infomaniak.com/livecast/dbmtv/playlist.m3u8
 https://mn-nl.mncdn.com/awraastv/awraastv_hd.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="FashionTVCzechSlovak.fr",FashionTV Czech&Slovak (450p) [Not 24/7]
 http://lb.streaming.sk/fashiontv/stream/playlist.m3u8
+#EXTINF:-1 tvg-id="France 2", France 2 (720p)
+http://tv.pro-ott.com:25443/live/HAJG6/k7bKPqg05y/15226.m3u8
 #EXTINF:-1 tvg-id="France24English.fr",France 24 English (1080p)
 http://n1.klowdtv.net/live2/france24_720p/chunks.m3u8
+#EXTINF:-1 tvg-id="France 3", France 3 (540p)
+http://ipro.webvps.de:25443/live/o1tEXUfCO0/QE4I5CeVb1/271306.m3u8
```

Comment:
x-stream codes links are not allowed

Reply:
![image](https://github.com/user-attachments/assets/53206ee8-ff61-465b-bf5b-22e57426897b)